workflows/methylseq.nf

```diff
@@ -19,12 +20,21 @@ def checkPathParamList = [
     params.bismark_index,
     params.known_splices,
 ]
-for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
-
-// Check mandatory parameters
-if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
 
+// Iterate over the list, checking each parameter
+for (param in checkPathParamList) { 
+    if (param) { 
+        file(param, checkIfExists: true) 
+    } 
+}
 
+// Validate the mandatory parameter 'input'
+if (params.input) { 
+    ch_input = file(params.input) 
+} else { 
+    // If 'input' is not specified, terminate the program with an error message
+    exit 1, 'Input samplesheet not specified!' 
+}
```

Comment:
Is this changing from the template? It's probably just easier to leave it as is.

Reply:
got ya